pkg/capauth/creds.go

```diff
@@ -18,15 +18,18 @@ import (
 )
 
 const (
-	ServCert      = "/etc/opt/vault/certs/serv_cert.pem"
-	ServCertLocal = "./serv_cert.pem"
-	ServKey       = "/etc/opt/vault/certs/serv_key.pem"
+	ServCert           = "/etc/opt/vault/certs/serv_cert.pem"
+	ServCertPrefixPath = "/etc/opt/vault/certs/"
+	ServCertLocal      = "./serv_cert.pem"
+	ServKey            = "/etc/opt/vault/certs/serv_key.pem"
 )
 
 var MashupCertPool *x509.CertPool
 
-func ReadServerCert() ([]byte, error) {
-	if _, err := os.Stat(ServCert); err == nil {
+func ReadServerCert(certName string) ([]byte, error) {
+	if _, err := os.Stat(ServCertPrefixPath + certName); err == nil { //To support &certName=??
```

Comment:
Can you len check certName first?  I'd rather not use it at all if certName length is 0.

Reply:
If certName is provided (via that url parameter) and it's invalid, it should error here rather than falling back to pulling the default cert.